src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -180,26 +169,18 @@ public bool UsingDefaultValue
             {
                 usingDefaultValue = value;
                 RaisePropertyChanged("UsingDefaultValue");
-                RaisePropertyChanged("ToolTipContent");
+                RaisePropertyChanged("ToolTip");
             }
         }
 
         /// <summary>
-        /// Controls whether the Use Default Value option is available.
-        /// </summary>
-        [JsonIgnore]
-        public bool DefaultValueEnabled
```

Comment:
API Change: `PortModel.DefaultValueEnabled` has been removed. Please use `PortModel.DefaultValue == null` to check whether a port has a default value.